app/Http/Controllers/ProfileController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Profile;
+use App\Models\Post;
+use App\Models\User;
+
+class ProfileController extends Controller
+{
+    /**
+     * Display my profile.
+     *
+     * @param Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function myProfile(Request $request)
+    {
+        $user = auth()->user();
+
+        $posts = Post::with('user')->where('user_id', $user->id)->orderBy('created_at', 'desc')->paginate(config('home.page.number'));
```

Comment:
```suggestion
        $posts = Post::with('user')
            ->where('user_id', $user->id)
            ->orderBy('created_at', 'desc')
            ->paginate(config('home.page.number'));
```
Nó bị dài quá thì viết xuống dòng cho đẹp e ơi